pkg/backend/display/testdata/not-truncated/template-body.json.interactive-100x80-cooked.stdout.txt

```diff
@@ -2770,1230 +2770,1230 @@
+ <{%reset%}>  <{%reset%}>     ├─ eks:index:ServiceRole             cluster-eksRole                            <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     │  ├─ aws:iam:Role                   cluster-eksRole-role                       <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     │  ├─ aws:iam:RolePolicyAttachment   cluster-eksRole-4b490823                   <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     │  └─ aws:iam:RolePolicyAttachment   cluster-eksRole-90eb1c99                   <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     ├─ eks:index:RandomSuffix            cluster-cfnStackName                       <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     ├─ aws:ec2:SecurityGroup             cluster-eksClusterSecurityGroup            <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     ├─ aws:ec2:SecurityGroupRule         cluster-eksClusterInternetEgressRule       <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     ├─ aws:iam:InstanceProfile           cluster-instanceProfile                    <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     ├─ aws:eks:Cluster                   cluster-eksCluster                         <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     ├─ eks:index:VpcCni                  cluster-vpc-cni                            <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     ├─ aws:ec2:SecurityGroup             cluster-nodeSecurityGroup                  <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     ├─ pulumi:providers:kubernetes       cluster-eks-k8s                            <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     ├─ aws:ec2:SecurityGroupRule         cluster-eksExtApiServerClusterIngressRule  <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     ├─ aws:ec2:SecurityGroupRule         cluster-eksClusterIngressRule              <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     ├─ aws:ec2:SecurityGroupRule         cluster-eksNodeInternetEgressRule          <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     ├─ aws:ec2:SecurityGroupRule         cluster-eksNodeIngressRule                 <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     ├─ aws:ec2:SecurityGroupRule         cluster-eksNodeClusterIngressRule          <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     ├─ kubernetes:core/v1:ConfigMap      cluster-nodeAccess                         <{%reset%}><{%reset%}>           
+ <{%reset%}>  <{%reset%}>     ├─ aws:ec2:LaunchConfiguration       cluster-nodeLaunchConfiguration            <{%reset%}><{%reset%}>           
+ <{%bold%}><{%fg 3%}>~ <{%reset%}>     └─ aws:cloudformation:Stack          cluster-nodes                              <{%bold%}><{%fg 3%}>updating<{%reset%}><{%bold%}><{%fg 3%}><{%reset%}>   
```

Comment:
Kudos: Having snapshot based tests for display logic make reviewing these PRs much easier. 👍